CIP-XXXX/README.md

```diff
@@ -0,0 +1,447 @@
+---
+CIP: 88
+Title: Hybrid Token Policy Registration
+Category: Tokens
+Status: Proposed
+Authors:
+    - Adam Dean <adam@crypto2099.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/467
+Created: 2023-02-27
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Currently, token projects (NFT or FT) have no mechanism to register the intent, details, or feature set of their minting
+policy on-chain. This CIP will aim to create a method that is backwards compatible and enable projects to declare, and
+update over time, the supported feature set and metadata details pertaining to their tokens.
+
+This CIP will aim to make use of a hybrid (on and off-chain) information schema to enable maximum flexibility and 
+adaptability as new and novel use cases for native assets (NA) expand and grow over time.
+
+## Motivation
+
+This CIP was borne out of a distaste for the lack of on-chain token policy intent registration that has been cited as
+both a centralization and security concern at various points over the preceding two years of NA history on Cardano.
+
+### Example 1: The Cardano Token Registry
+
+Many Fungible Token (FT) projects require special treatment of their NAs such as decimal places for proper display and
+formatting, project information, and token logo. As it stands, these projects must currently register via a GitHub
+repository ([Cardano Token Registry](https://github.com/cardano-foundation/cardano-token-registry)) in order to have
+their token properly appear in wallets. This GitHub repository is currently managed by the Cardano Foundation (CF).
+While there is no reason to believe that the CF would take any malicious or nefarious action, forcing token projects to
+register in this fashion introduces a point of centralization, _potential_ gate keeping, and ultimately reliance on the
+interaction of a 3rd party (the human at CF responsible for merging pull requests) in order for projects to register, or
+update, their information.
```

Comment:
As a side-note: the intent was never for the CF to be the sole actor to provide a token registry. When originally envisioned, the entire solution was meant to be run and provided by various actors. The CF would only be one of the possible actors. Yet, as time now shows, there hasn't been any actor stepping up to provide their own token metadata server and wallets implementators haven't given this flexibility (of choosing their own source of data) to users.